### ISSUE TITLE
Error debugging GeoDjango queries

### ISSUE THREAD

**Issue by rodrigoolmo** [jazzband/django-debug-toolbar#423]
When debugging queries that use GeoDjango they are parsed like normal strings.
Example:

```
params = [u"ST_GeomFromEWKB('\\x0101000020e6100000555458045a9d16c010470d5cd47c4440'::bytea)"]
sql = 'SELECT (ST_distance_sphere("location"."point",%s)) AS "distance" FROM "location"'
```

What it should execute:

```
SELECT (ST_distance_sphere("location"."point", ST_GeomFromEWKB('\x0101000020e6100000555458045a9d16c010470d5cd47c4440'::bytea))) AS "distance" FROM "location"
```

What it really executes:

```
SELECT (ST_distance_sphere("location"."point", 'ST_GeomFromEWKB(''\x0101000020e6100000555458045a9d16c010470d5cd47c4440''::bytea)')) AS "distance" FROM "location"
```

I am using django-debug-toolbar version 0.9.4, Django 1.4 and PostgreSQL with PostGIS.

**Comment by jperelli** [comment]
I have the same problem, using `django==2.1.4` and `django-debug-toolbar==1.11`

**Comment by tim-schilling** [comment]
Reopend as per #1547

**Comment by jperelli** [comment]
I used this today and I was very surprised that just worked fine!

I couldn't find it in the changelog but it seems you landed the fix in https://github.com/jazzband/django-debug-toolbar/pull/1426

I think you can close this one @tim-schilling / @aaugustin 

Thank you very much guys!!

**Comment by tim-schilling** [comment]
@jperelli that change was reverted as it mutates data going into your database. You should upgrade versions of the toolbar.

**Comment by jperelli** [comment]
@tim-schilling You are right, sorry abot that. I can see it now in a query that uses a geom as a parameter
![image](https://user-images.githubusercontent.com/1418446/185187520-a2fd3904-ccdb-4431-9447-7a67caeb3e2d.png)
![image](https://user-images.githubusercontent.com/1418446/185187614-2439fd27-93d1-48b9-a6e9-46d6a066eba4.png)
![image](https://user-images.githubusercontent.com/1418446/185187822-c1d1164c-d019-4c31-bffd-ac0494b91e56.png)
:+1:
